test/unit/comments.spec.ts

```diff
@@ -0,0 +1,37 @@
+import * as util from "../util";
+
+test("JSDoc is copied on a function", () => {
+    const builder = util.testModule`
+        /**
+         * This is a function comment.
+         * It has multiple lines.
```

Comment:
Could you also add a test (with snapshot) for a jsdoc comment with jsdoc tags like `@param` and `@returns`